sunpy/timeseries/sources/goes.py

```diff
@@ -235,8 +235,12 @@ def _parse_netcdf(filepath):
             flux_name_b = flux_name_a.replace("a", "b")
             xrsa = np.array(h5nc[flux_name_a])
             xrsb = np.array(h5nc[flux_name_b])
-            xrsa_quality = np.array(h5nc[flux_name_a.replace("flux", "flags")])
-            xrsb_quality = np.array(h5nc[flux_name_b.replace("flux", "flags")])
+            flux_flag_a = h5nc.variables.get("a_flags") or h5nc.variables.get("xrsa_flags") or h5nc.variables.get("xrsa_flag")
```

Comment:
That is quite annoying that they vary at least 3 ways.

Reply:
I know the stuff has been running for 60 years or something so changes are excused but there is so much stuff like this with GOES which make it a pain.

Reply:
I think we should perhaps think about this some more and maybe have specific 1m/ 2s/background timeseries classes or something

Reply:
Even within the same data series they are inconsistent with file names/column names/metadata so while it might be an idea, it doesn't make this kind of thing any tidier.

As an aside I have a scraper for 1m data but its ugly (secondary client rather than using attrs to select series) so haven't tried to put it in sunpy yet.